shudder/queue.py

```diff
@@ -19,37 +19,38 @@
 """
 import json
 
-from boto.utils import get_instance_metadata
-import boto.sns as sns
-import boto.sqs as sqs
+import boto3
 
 from shudder.config import CONFIG
+import shudder.metadata as metadata
 
 
-INSTANCE_ID = get_instance_metadata()['instance-id']
+INSTANCE_ID = metadata.get_instance_id()
```

Comment:
See various comments above about how boto3 doesn't have this function that boto2 does, and how I added it to metadata.py